src/components/MapView/Analyser/AnalysisTable/index.tsx

```diff
@@ -19,6 +21,8 @@ import { Column } from '../../../../utils/analysis-utils';
 function AnalysisTable({ classes, tableData, columns }: AnalysisTableProps) {
   const [page, setPage] = useState(0);
   const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [sortColumn, setSortColumn] = useState<Column['id']>();
+  const [sortDirection, setSortDirection] = useState<'asc' | 'desc'>('asc');
```

Comment:
Nitpick: For something with 2 states I'd prefer a boolean (isAscending) so less can go wrong (string can be set to something random even if it's protected by TS). This is completely up to personal preference however.

Reply:
agreed. done